### DISCUSSION TITLE
Upload datr and time

### DISCUSSION THREAD

**Discussion by AlanSanin** [ClassicDIY/SkyeTracker#47] [General]
How to upload date and time on the skye tracker app

**Comment by AlanSanin** [comment]
I have error on the date an time for the skye tracker app

**Reply by KaHu85** [reply]
That works on my Android Phone with the APP, But i have Problems with the Longitude and Latitude...
I go to Date and Time and klick Upload, only after a Power Failure Date and Time are Gone...

**Reply by AlanSanin** [reply]
what kind of power failure?
I click upload date and time but nothing happen

**Reply by AlanSanin** [reply]
update! the app stop working when the location change. Is my rtc module the problem here?

**Comment by graham22** [comment]
Starting with a fresh ESP32 WROOM 32D, I installed the firmware as described in the wiki, configured the connection to wifi, accessed the configuration page by browsing to the IP address of the ESP32, configured the MQTT broker.
I installed the android app on a Pixel 2 device with bluetooth enabled.
Everything seemed to work, the esp datetime was set by the NTP service.
I used the MQTT.fx tool to confirm the MQTT data was being published to the broker



![MQTT](https://github.com/ClassicDIY/SkyeTracker/assets/5341261/159383b9-3749-4ce8-bc0e-636b2e374fdd)
![Screenshot (Jan 23, 2024 13 01 56)](https://github.com/ClassicDIY/SkyeTracker/assets/5341261/0f4fb5ee-06a6-4300-b1c4-c113d6a18511)
![broker](https://github.com/ClassicDIY/SkyeTracker/assets/5341261/a14ab066-3d5d-4cda-965b-3a1c32b84e61)

**Reply by graham22** [reply]
I did find a problem with the android app, 
a new permission must now be declared in the manifest to access lat/lon location
will post an update soon...

**Reply by AlanSanin** [reply]
Understood and thank you very much for your reply
